### ISSUE TITLE
Download links are not working

### ISSUE THREAD

**Issue by Mardiniii** [jigish/slate#532]
Hey @jigish, it looks like the links to get the installers or running the installation command from the terminal are not working? 

Is there a way I can help to fix them?

**Comment by Jannertje** [comment]
I cannot vouch for this source, but I was able to find a tarball on MacUpdate:
https://www.macupdate.com/app/mac/45215/slate/download/secure
If anyone has the (confirmed) 1.0.25 Slate.app and could verify that the SHA-1 checksum is `45f9d79216763b7da62c093595aa6dae10592dba` (check this through `shasum -a 1 /Applications/Slate.app/Contents/MacOS/Slate`), that would be helpful.

**Comment by dmdre** [comment]
Yes, it is the checksum of the file : https://github.com/jigish/slate/blob/master/build/Release/Slate.dmg, 
But you must download it from the repo now as it is available 👍
